bespin_k8s_gcloud_cluster/README.md

```diff
@@ -0,0 +1,46 @@
+# bespin\_k8s\_gcloud\_cluster
+
+Ansible role to create and setup a google cloud k8s cluster and namespace that allows running jobs via bespin.
+
+## Requirements
+
+Uses the [bespin_k8s_namespace](../bespin_k8s_namespace/README.md) ansible role, so install it's requirements.
+
+Command line tool requirements:
+- gcloud has been installed (including [beta components](https://cloud.google.com/sdk/gcloud/reference/components/install)) and is authenticated to google cloud
+- kubectl has been installed
+- ansible has been installed
```

Comment:
The PR description mentions that a development version of ansible is required, so that should be noted here too.